network/version.go

```diff
@@ -25,6 +25,7 @@ const (
 	Version15                 // ???
 	Version16                 // ???
 	Version17                 // ???
+	Version18                 // FVM M2.1
```

Comment:
I'd suggest adding a couple more versions here just to have them ready for buildernet.

Also can you add the names for Versions 15-17 please?

15 is OhSnap, 16 is Skyr, 17 is Shark

Reply:
sure

Reply:
done.